src/myst.ts

```diff
@@ -56,9 +56,11 @@ export function markdownParse(text: string): Root {
       }
     })
     .runSync(mdast as any);
-  // Lift children out of blocks for the next step
-  // We are working here as one cell at a time
-  liftChildren(mdast, 'block');
+  if (inNotebook) {
+    // If in the notebook, lift children out of blocks for the next step
+    // We are working here as one cell at a time
+    liftChildren(mdast, 'block');
+  }
```

Comment:
The structure of the markdown should stay in the blocks that the markdown gave, which is not the case for cells, which MUST lift children of the blocks so that we can target and identify them.